ci/terraform/oidc/api-gateway.tf

```diff
@@ -133,6 +133,7 @@ resource "aws_api_gateway_deployment" "deployment" {
       module.ipv-capacity.method_trigger_value,
       var.doc_app_api_enabled ? module.doc-app-callback[0].integration_trigger_value : null,
       var.doc_app_api_enabled ? module.doc-app-callback[0].method_trigger_value : null,
+      var.use_robots_txt ? aws_api_gateway_integration_response.robots_txt_integration_response : null,
```

Comment:
I think the response will need a `[0]` on it

Reply:
Because there's a count

Reply:
thanks updated